osmomath/math_test.go

```diff
@@ -37,6 +37,21 @@ func TestPowApprox(t *testing.T) {
 		expectedDec.Sub(s).Abs().LTE(powPrecision),
 		"expected value & actual value's difference should less than precision",
 	)
+
+	base, err = sdk.NewDecFromStr("0.8")
+	require.NoError(t, err)
+	exp = sdk.ZeroDec()
+	require.NoError(t, err)
+
+	s = PowApprox(base, exp, powPrecision)
+	expectedDec = sdk.OneDec()
+	require.NoError(t, err)
+
+	require.True(
+		t,
+		expectedDec.Sub(s).Abs().LTE(powPrecision),
+		"expected value & actual value's difference should less than precision",
+	)
```

Comment:
Commenting to create an issue for post v10 work (not in this PR):

Make `TestPowApprox` and `TestPow` table-driven and add more test cases